src/libpanic_unwind/lib.rs

```diff
@@ -39,6 +39,10 @@ cfg_if::cfg_if! {
     if #[cfg(miri)] {
         #[path = "miri.rs"]
         mod imp;
+        // On MSVC we need the SEH lang items as well...
+        #[cfg(all(target_env = "msvc", not(target_arch = "aarch64")))]
```

Comment:
Could you add a comment stating that this should match the corresponding 'normal' `#[cfg(all(target_env = "msvc", not(target_arch = "aarch64")))]` below?